<launch>
    <arg name="state_ai_name" default="state_ai" />
    <arg name="is_simulation" default="false" />
    <node name="$(arg state_ai_name)" pkg="capra_ai" type="$(arg state_ai_name).py" output="screen" >
        <param name="is_simulation" value="$(arg is_simulation)" />
    </node>
    <node pkg="capra_2dnav" type="obstacle_generator.py" name="obstacle_generator" output="screen" />
</launch>